dashboard/apps/prototype/migrations/0001_initial.py

```diff
@@ -1,82 +0,0 @@
-# -*- coding: utf-8 -*-
```

Comment:
good move to remove this for now. might be worth adding the directory in `.gitignore` for the time being, so they don't accidentally get added in again.